site/docs/relations/logical_relations.md

```diff
@@ -261,14 +261,35 @@ The set operation encompasses several set-level operations that support combinin
 
 ### Set Operation Types
 
-| Property                | Description                                                  |
-| ----------------------- | ------------------------------------------------------------ |
-| Minus (Primary)         | Returns the primary input excluding any matching records from secondary inputs. |
-| Minus (Multiset)        | Returns the primary input minus any records that are included in all sets. |
-| Intersection (Primary)  | Returns all rows primary rows that intersect at least one secondary input. |
-| Intersection (Multiset) | Returns all rows that intersect at least one record from each secondary inputs. |
-| Union Distinct          | Returns all the records from each set, removing any rows that are duplicated (within or across sets). |
-| Union All               | Returns all records from each set, allowing duplicates.      |
+The set operation type determines both the records that are emitted and the type of the output record.
+
+| Property                | Description                                                                                                   | Output Shape
+| ----------------------- | ------------------------------------------------------------------------------------------------------------- | ----------------------------- |
+| Minus (Primary)         | Returns all records from the primary input excluding any matching records from secondary inputs.              | The same as the primary input
+| Minus (Multiset)        | Returns all records from the primary input excluding any records that are included in *all* secondary inputs. | The same as the primary input
+| Intersection (Primary)  | Returns all records from the primary input that match at least one record from *any* secondary inputs.        | If a field is nullabe in in the primary input and in any of the secondary inputs, is nullable in the output.
+| Intersection (Multiset) | Returns all records from the primary input that match at least one record from *all* secondary inputs.        | If a field is non-null in any of the inputs, it is non-null in the output.
+| Union Distinct          | Returns all the records from each set, removing any rows that are duplicated (within or across sets).         | If a field is nullable in any of the inputs, it is nullable in the output.
+| Union All               | Returns all records from each set, allowing duplicates.                                                       | If a field is nullable in any of the inputs, it is nullable in the output |
```

Comment:
End with a period for consistency?  (and above)

Reply:
Now that you've pointed it out I cannot unsee it. Updated.